docs/changes/utils/12891.feature.rst

```diff
@@ -0,0 +1,5 @@
+* Added the ``where`` keyword argument to ``astropy.utils.masked.MaskedNDArray.mean()``.
+* Added the ``where`` keyword argument to ``astropy.utils.masked.MaskedNDArray.var()``.
+* Added the ``where`` keyword argument to ``astropy.utils.masked.MaskedNDArray.std()``.
+* Added the ``where`` keyword argument to ``astropy.utils.masked.MaskedNDArray.any()``.
+* Added the ``where`` keyword argument to ``astropy.utils.masked.MaskedNDArray.all()``.
```

Comment:
Are we allowed to use the same filename, `12891.feature.rst`, as long as it's in a different path?

Reply:
I think so... but why can't this be one sentence like this?

*Added there `where` keyword argument to X, Y, ..., and Z*

Reply:
Ok, so the preferred way to do this is two files, with one sentence in each?

Reply:
I think so but @saimn can confirm.